easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -508,15 +514,16 @@ def compose_install_command(self, prefix, extrapath=None, installopts=None):
         if extrapath:
             cmd.append(extrapath)
 
-        if self._should_unpack_source():
-            # specify current directory
-            loc = '.'
-        else:
-            # for extensions, self.src specifies the location of the source file
-            # otherwise, self.src is a list of dicts, one element per source file
-            if isinstance(self.src, string_type):
+        loc = self.cfg.get('install_src')
+        if not loc:
```

Comment:
`if loc is None` is more correct

Reply:
True, but we don't want an empty string here either.